### PR TITLE
Change times and add people

### PR DESCRIPTION
**Pull request by libbyschuknight**
stand up to 830am
stand down back to 245pm

and addition of Nat and Charlotte with them not on the days they aren't here

### CODE REVIEW

File: src/shared/config.js [hunk 1]
```diff
@@ -55,10 +55,11 @@ module.exports = [
   {
     name: 'weka',
     memeOfTheWeek: false,
-    standupCaptains: ['libbyschuknight', 'sandemchad', 'timothyjohn2015', 'fourseven', 'eileen', 'dave'],
+    standupCaptains: ['libbyschuknight', 'sandemchad', 'timothyjohn2015', 'fourseven', 'eileen', 'dave', 'nat', 'Charlotte' ],
```
Comment (timothyjohn2015) [inline src/shared/config.js:58]: So we can call Natalie Nat? Is she ok with nick names?

Comment (timothyjohn2015) [reply src/shared/config.js:58]: Also can we remove Chad from here please:
```suggestion
    standupCaptains: ['libbyschuknight', 'timothyjohn2015', 'fourseven', 'eileen', 'dave', 'nat', 'Charlotte' ],
```

File: src/shared/users.js [hunk 2]
```diff
@@ -32,4 +32,5 @@ module.exports.users = {
   eileen: '<@ULHKU7MQW>',  // Eileen
   dave: '<@U01KNSCGVFW>',  // Dave Keyes
   abbie: '<@U035HE79WFP>', // Abbie
+  charlotte: '<@U035HE79WFP>', // Charlotte
```
Comment (timothyjohn2015) [inline src/shared/users.js:35]: Where do we get this value?

**Review by timothyjohn2015 — APPROVED** [review]
LGTM just a few questions

File: src/shared/config.js (hunk 1 above)
Comment (libbyschuknight) [reply src/shared/config.js:58]: Oh thought I had removed Chad. Nat goes by Nat. See her slack handle, it is `@nat`. but am also double checking with her...

File: src/shared/users.js (hunk 2 above)
Comment (libbyschuknight) [reply src/shared/users.js:35]: No idea, don't even know what it is for.

File: src/shared/config.js (hunk 1 above)
Comment (libbyschuknight) [reply src/shared/config.js:58]: Yep she is happy with Nat.

Comment (libbyschuknight) [reply src/shared/config.js:58]: I think she wants to change her name to Mat though 😂

File: src/shared/users.js (hunk 2 above)
Comment (timothyjohn2015) [reply src/shared/users.js:28]: Oh I see, you just copied Abbies one. I think they're just stubs? But they may need to be unique..

Comment (libbyschuknight) [reply src/shared/users.js:28]: we will find out I guess

Comment (BenS-SP) [reply src/shared/users.js:28]: That's the slack userid. Got to the persons profile in slack, click the 3 dots (more) and Copy Member ID is an option. That's where you get it from

Comment (timothyjohn2015) [reply src/shared/users.js:28]: Oh yep just re-read the comment. Makes sense. So this will need updating then

Comment (libbyschuknight) [reply src/shared/users.js:28]: sweet thanks, will do that shortly

Comment (libbyschuknight) [reply src/shared/users.js:28]: Oh doh!

`// This maps Github usernames -> Slack userIds`

Comment (libbyschuknight) [reply src/shared/users.js:28]: trying to do it too fast! 😂

**Pull request merged**